deployer/src/handlers/mod.rs

```diff
@@ -621,6 +623,12 @@ pub async fn start_deployment(
         };
         deployment_manager.run_push(built).await;
 
+        let event =
+            async_posthog::Event::new("shuttle_api_start_deployment".to_string(), account_name);
+        if let Err(err) = deployment_manager.posthog_client().capture(event).await {
+            error!(error = %err, "failed to send event to posthog")
+        };
```

Comment:
nit: let this be done in a tokio task to not disrupt the execution of this thread and delay the deployment.